### PR TITLE
web: Get fees dynamically

### LINKED ISSUE #209: Get fees dynamically
## Is your feature request related to a problem? Please describe.
I'm always frustrated when we have hardcoded values for dynamic things.

## Describe the solution you'd like
Use REST to get fee values.

## Additional context
#197. Blocked by https://github.com/nspcc-dev/neofs-rest-gw/issues/198.

### PR DESCRIPTION
**Pull request by mike-petrov**
for namedContainerFee & withdrawalFee

closes #209

### CODE REVIEW

**Comment by mike-petrov** [comment]
This parameters for testnet (https://rest.t5.fs.neo.org/v1/network-info):

Screenshot 2024-08-08 at 22 02 06

Screenshot 2024-08-08 at 22 02 43

Should we use namedContainerFee for creation container and withdrawalFee for withdrawal?

File: src/App.js [hunk 1]
```diff
@@ -1083,7 +1090,7 @@ export const App = () => {
 									Container creation is a paid operation, make sure you have sufficient NeoFS chain balance
 								</Notification>
 							)}
-							<Heading className="input_caption">Container cost is 0.0070000014 GAS for mainnet now.</Heading>
+							<Heading className="input_caption">{`Container cost is ${networkInfo ? networkInfo.namedContainerFee * 0.00000001 : '-'} GAS for mainnet now.`}</Heading>
```
Comment (roman-khimov) [inline src/App.js:1093]: It's _both_ `namedContainerFee` and `containerFee`. And it's in picoGAS, 10^-12.

Comment (roman-khimov) [reply src/App.js:1093]: Ah, and it's also `×7`, paid per node.

File: src/App.js [hunk 2]
```diff
@@ -1717,7 +1724,7 @@ export const App = () => {
 							/>
 						</div>
 						<Heading align="center" size={5} weight="bold">{`Withdraw from NeoFS to ${activeNet}`}</Heading>
-						<Heading className="input_caption" style={{ maxWidth: 320 }}>Withdrawing requires a fee to be paid, currently it's 7 GAS. It will be reduced once the <a href="https://github.com/neo-project/neo/issues/1573" target="_blank" rel="noopener noreferrer" alt="neofs-node">notary subsystem</a> is implemented in Neo</Heading>
+						<Heading className="input_caption" style={{ maxWidth: 310 }}>{`Withdrawing requires a fee to be paid, currently it's ${networkInfo ? networkInfo.withdrawalFee * 0.00000001 : '-'} GAS.`} It will be reduced once the <a href="https://github.com/neo-project/neo/issues/1573" target="_blank" rel="noopener noreferrer" alt="neofs-node">notary subsystem</a> is implemented in Neo</Heading>
```
Comment (roman-khimov) [inline src/App.js:1727]: This one is also ×7 (but 10^-8, correct wrt this now).

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**